docs/setup-cluster/deploy-cluster/slurm/upgrade-on-hpc.rst

```diff
@@ -0,0 +1,39 @@
+.. _upgrade-on-hpc:
+
+#################################
+ Upgrade Determined on Slurm/PBS
+#################################
+
+This procedure describes how to upgrade Determined on an HPC cluster managed by the Slurm or PBS
+workload managers. Use this procedure when an earlier version of Determined is installed,
+configured, and functioning properly.
+
+#. Review the latest :doc:`/setup-cluster/deploy-cluster/slurm/slurm-requirements` and ensure all
+   dependencies have been met.
+
+#. Upgrade the launcher.
+
+   For an example RPM-based installation, run:
+
+   .. code:: bash
+
+      sudo rpm -Uv hpe-hpc-launcher-<version>.rpm
+
+   On Debian distributions, run:
+
+   .. code:: bash
+
+      sudo apt install ./hpe-hpc-launcher-<version>.deb
+
+   The upgrade automatically updates and restarts the ``systemd`` ``launcher`` service.
+
+#. Upgrade the on-premises Determined master component (not including the Determined agent) as
+   described in the :ref:`upgrades` document.
+
+   The upgrade does not automatically update the Determine master service. Reload the ``systemd``
+   configuration and restart the Determine master service with the following commands.
+
```

Comment:
The upgrade does not automatically update the Determine master service. Reload the ``systemd``
   configuration and restart the Determine master service with the following commands:

Reply:
Hi Tara, Ilia, and all the others.
My apologies, but as Ilia said, it seems that something strange happened to this branch with the command: `git rebase main`. I propose the following, If all agree, I could prepare another branch with the same  code changes starting from a clean `determined` repo clone, then we can delete this branch, and PR the new one.

Sorry again for this issue, but I really don't get why this has happened, anyway we have the solution.

I might proceed, if ok for everyone. Your PoV and/or any other approach is more than welcome.